tests/pdata_test.c

```diff
@@ -0,0 +1,258 @@
+#define UNICODE 1
+#define _UNICODE 1
+
+#include <windows.h>
+#include <stdio.h>
+#include <dbghelp.h>
+
+#pragma comment(lib, "dbghelp.lib")
+
+// build with
+// ~/cross/bin/aarch64-w64-mingw32-gcc test-pdata.c -ldbghelp -o test-pdata.exe
+//
+// View pdata
+// ~/cross/bin/aarch64-w64-mingw32-objdump -s -j .pdata test-pdata.exe
+
+#define STACK_MAX 16
+#define SYMBOL_MAXLEN 256
+
+static void print_symbol(HANDLE hp, DWORD64 offset)
+{
+	IMAGEHLP_MODULEW64 module;
+	memset(&module, 0, sizeof(module));
+	module.SizeOfStruct = sizeof(module);
+	SymGetModuleInfoW64(hp, offset, &module);
+
+	if (wcslen(module.ModuleName) > 0)
+	{
+		//wprintf(L"%-8s", module.ModuleName);
+		wprintf(module.ModuleName);
+	}
+
+	BYTE buffer[sizeof(IMAGEHLP_SYMBOL64) + SYMBOL_MAXLEN];
+	memset(buffer, 0, sizeof(buffer));
+
+	const char* symbolName = "";
+	DWORD64 dwDisplacement = 0;
+	IMAGEHLP_SYMBOL64* p = (IMAGEHLP_SYMBOL64*)buffer;
+	p->SizeOfStruct = sizeof(IMAGEHLP_SYMBOL64);
+	p->MaxNameLength = SYMBOL_MAXLEN;
+
+	if (SymGetSymFromAddr64(hp, offset, &dwDisplacement, p))
+	{
+		printf(" %s+0x%.04llX", p->Name, dwDisplacement);
```

Comment:
This test is heavily versbose, we should disable printing the standard output, at least for this test.

Reply:
It has been commented.